components/identities/actions/ActiveSubscriptionCard.tsx

```diff
@@ -1,12 +1,35 @@
 import React from "react";
 import styles from "../../../styles/components/identityMenu.module.css";
+import { timestampToReadableDate } from "@/utils/dateService";
+import { Identity } from "@/utils/apiWrappers/identity";
+
+interface ActiveSubscriptionCardProps {
+  identity: Identity;
+}
+
+function ActiveSubscriptionCard({ identity }: ActiveSubscriptionCardProps) {
+  const cancelSubscription = () => {

```

Comment:
No you need to use this instead:
```tsx
 const { sendAsync: disableRenewal, data: disableRenewalData } =
    useSendTransaction({
      calls: disableRenewalCalldata,
    });
```

All other necessary code and dependencies are here https://github.com/lfglabs-dev/app.starknet.id/blob/2ce90db5ce88e83c01a8f8fa8741692065bf642f/components/identities/actions/identityActions.tsx